pkg/transport/http/listener_options_test.go

```diff
@@ -0,0 +1,89 @@
+package http
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+	"time"
+)
+
+func Test_NewListenerOptions(t *testing.T) {
```

Comment:
Let's keep the test names consistent? I have not yet used underscores in test names inside this project.